x-pack/legacy/plugins/apm/public/components/shared/MetadataTable/__test__/MetadataTable.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import 'jest-dom/extend-expect';
+import { render, cleanup } from 'react-testing-library';
+import { MetadataTable } from '..';
+import {
+  expectTextsInDocument,
+  expectTextsNotInDocument
+} from '../../../../utils/testHelpers';
+
+describe('MetadataTable', () => {
+  afterEach(cleanup);
+  describe('required sections', () => {
+    it('shows message if no data is available', () => {
+      const sections = [
+        {
+          key: 'foo',
+          label: 'Foo',
+          required: true
+        }
+      ];
+      // @ts-ignore
+      const output = render(<MetadataTable item={{}} sections={sections} />);
```

Comment:
Better than `@ts-ignore` is to cast values so the intention is clear:
```suggestion
      const output = render(<MetadataTable item={{} as Transaction} sections={sections} />);
```